test-app/tests/integration/components/input-field-test.gts

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+import { fillIn, find, render, setupOnerror } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import InputField from '@crowdstrike/ember-toucan-core/components/form/input-field';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+
+module('Integration | Component | InputField', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template>
+      <InputField
+        @label="Label"
+        type="text"
+        data-input
+        />
+    </template>);
+
+    const label = '[data-label]'
+    const input = '[data-input]';
+    const inputId = find(input)?.getAttribute('id') || '';
```

Comment:
When the input has an ID of an empty string (and having `<label for="">`), then I think this would make this test pass. Better to assert also for the existence of a proper ID:

```suggestion
    const inputId = find(input)?.getAttribute('id');
    
    assert.ok(inputId, 'Expected to have ID');
```

Reply:
<img width="1023" alt="Screen Shot 2023-02-28 at 12 18 34 PM" src="https://user-images.githubusercontent.com/771170/221928523-a0c2fa35-13ed-4ddb-a089-1d46207349c7.png">

The linter was complaining about the line change on 24 but I added the other suggestion - thank you!